feature-find-meal/src/main/java/com/kinandcarta/feature/find/meal/viewmodel/DisplayMealsViewModel.kt

```diff
@@ -43,62 +46,62 @@ class FindMealViewModel @ViewModelInject constructor(
     private val locationCallback = object : LocationCallback() {
         override fun onLocationResult(locationResult: LocationResult?) {
             lastLocation = locationResult?.lastLocation
-            updateMeals()
+            updateMeals("onLocationResult")
```

Comment:
maybe we can put this string in a constant?

Reply:
This param has been removed so no need :) it was used for bug fixing! Oopssss